nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,8 @@
+const { response } = require("express");
+const express = require("express");
+const app = express();
+const numbersRouter = require('./numbersRouter');
+
+app.get("/", (request, response) => response.send("nodejs week2 homework"));
+
+app.use('/numbers', numbersRouter);
```

Comment:
You are missing app.listen from this file. Without that, the server is not going to 'listen' for the requests. 
app.listen(3000, () => console.log(`Calculator:listening on port 3000`));

Reply:
Thank you for the great comments. I have added the app. listen part :)